src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck is working");
```

Comment:
The message 'Truck is working' should be more informative. According to the checklist, it should indicate the type of Machine that is working. Consider including the class name in the message.